src/settings/LocationInstitutions.js

```diff
@@ -63,15 +74,34 @@ class LocationInstitutions extends React.Component {
    * will be stale if they change between unmounting/remounting.
    */
   componentDidMount() {
-    this.props.mutator.locationsPerInstitution.reset();
```

Comment:
Should we remove `locationsPerInstitution` from mutator object on line 23?

Reply:
Thanks! I missed that. Already fixed

Reply:
@artem-blazhko please take a look at fixed changes